_data/ball/general.yaml

```diff
@@ -0,0 +1,17 @@
+events:
+  - name: 'CSS BALL'
+    year: '2022'
+    location: 'Fazeley Studios'
+    date: '27th April'
+    link: '/ball/2022'
+    image: '/assets/images/ball-22-dancing.jpg'
+    alt: 'Image of CSS Ball 2022'
+    attendees: '173'
+  - name: 'CSS BALL'
+    year: '2020'
+    location: 'The Malmaison'
+    date: '12th March'
+    link: '/ball/2020'
+    image: '/assets/images/ball-20-dancing.jpg'
+    alt: 'Image of CSS Ball 2020'
+    attendees: '130'
```

Comment:
Duplicating this information might be a bit of an issue, is it not possible to include it with or extend the normal [year].yml files?